packages/authentication/src/__tests__/acceptance/users/user.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/authentication
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+export interface User {
```

Comment:
Usually the `User` would be a model, can we define a model instead?

Reply:
The `user repository` used in similar, existing tests was a hard-coded list of JSON objects and didn't involve datasources and models...so I figured there is no need to create a model for my acceptance tests.